File: src/main/java/xyz/acevedosharp/entities/FamiliaDB.java
package xyz.acevedosharp.entities;

import javax.persistence.*;

@Entity
@Table(name = "familia", schema = "epic")
public class FamiliaDB {
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private @Id @Column(name = "familia_id") Integer familiaId;
    private @Column(name = "nombre") String nombre;

    public FamiliaDB() {
    }

    public FamiliaDB(Integer familiaId, String nombre) {
        this.familiaId = familiaId;
        this.nombre = nombre;
    }

    public Integer getFamiliaId() {
        return familiaId;
    }

    public String getNombre() {
        return nombre;
    }

    public void setNombre(String nombre) {
        this.nombre = nombre;
    }
}

File: src/main/kotlin/xyz/acevedosharp/persistence_layer/entities/FamiliaDB.kt
package xyz.acevedosharp.persistence_layer.entities

import javax.persistence.*

@Entity
@Table(name = "familia", schema = "epic")
class FamiliaDB(

    @Id @GeneratedValue(strategy = GenerationType.IDENTITY)
    @Column(name = "familia_id")
    var familiaId: Int? = null,

    @Column(name = "nombre")
    var nombre: String
)
